WTG.Analyzers/Analyzers/CompletedTask/CompletedTaskAnalyzer.cs

```diff
@@ -80,5 +85,18 @@ static void Analyze(SyntaxNodeAnalysisContext context, FileDetailCache cache)
 
 			context.ReportDiagnostic(Rules.CreatePreferCompletedTaskDiagnostic(invoke.GetLocation()));
 		}
+
+		static bool HasCompletedTask(Compilation compilation)
+		{
+			foreach (var symbol in compilation.GetTypeByMetadataName(typeof(System.Threading.Tasks.Task).FullName).GetMembers(nameof(System.Threading.Tasks.Task.CompletedTask)))
```

Comment:
Everywhere else, such as the code fix, these are string constants. I’d rather be consistent - and in the direction of strings, since we’re not reflecting into these well-known types and just using them to obtain their names.

Reply:
Using a string constant was my first though, but then I saw that you used this approach for [`Array.Empty<>()`](https://github.com/WiseTechGlobal/WTG.Analyzers/blob/4a6ffc7e0f713c46a5557bd231f7549d14042dd4/WTG.Analyzers/Analyzers/Array/ArrayAnalyzer.cs#L141) and decided to be consistent :)

Reply:
Hmmm, now why did I do that... 🤔